common/persistence/dataInterfaces.go

```diff
@@ -73,7 +73,12 @@ const (
 	TransferTaskTypeCancelExecution
 	TransferTaskTypeStartChildExecution
 	TransferTaskTypeSignalExecution
-	TransferTaskTypeReplicationTask
+)
+
+// Types of replication tasks
+const (
+	ReplicationTaskTypeHistory = iota
+	ReplicationTaskTypeHeartbeat
```

Comment:
is ReplicationTaskTypeHeartbeat in the next PR?

Reply:
Yes.  This is for future purpose.